components/raftstore/src/store/worker/read.rs

```diff
@@ -520,16 +524,26 @@ where
             // The local `ReadDelegate` is up to date
             Some(d) if !d.track_ver.any_new() => Some(Arc::clone(d)),
             _ => {
+                if self.get_must_track_version_changed_failpoint() {
+                    if let Some(d) = self.delegates.get(&region_id) {
+                        info!("must_track_version_changed: {}", d.track_ver.any_new());
+                    }
+                }
                 debug!("update local read delegate"; "region_id" => region_id);
                 self.metrics.rejected_by_cache_miss += 1;
 
+                let t = Instant::now();
+
                 let (meta_len, meta_reader) = {
                     let meta = self.store_meta.lock().unwrap();
                     (
                         meta.readers.len(),
                         meta.readers.get(&region_id).cloned().map(Arc::new),
                     )
                 };
+                if self.get_must_track_version_changed_failpoint() {
+                    info!("store meta lock acquire takes {:?}", t.saturating_elapsed());
+                }
```

Comment:
Can we use `slow_log` instead?

Reply:
This logic is not changed, but only fix for make clippy pass.